bundles/org.openhab.binding.zwavejs/src/3rdparty/java/com/google/gson/typeadapters/RuntimeTypeAdapterFactory.java

```diff
@@ -0,0 +1,320 @@
+/*
+ * Copyright (C) 2011 Google Inc.
```

Comment:
As you are using code from a third-party, I assume it should be mentioned in your NOTICE file.

Reply:
Have added it (the file itself also has copyright headers).